File: api/packages/v1/administrativo/services/tabelionato_protesto/go/tabelionato_protesto_dashboard_funil_service.py
from packages.v1.administrativo.actions.tabelionato_protesto.tabelionato_protesto_dashboard_funil_action import (
    TabelionatoProtestoDashboardFunilAction,
)


class TabelionatoProtestoDashboardFunilService:
    def execute(self):
        return TabelionatoProtestoDashboardFunilAction().execute()

File: app/src/packages/administrativo/services/TabelionatoProtestoDashboard/TabelionatoProtestoDashboardFunilService.ts
'use server';

import { TabelionatoProtestoDashboardFunilData } from '@/packages/administrativo/data/TabelionatoProtestoDashboard/TabelionatoProtestoDashboardFunilData';
import { withClientErrorHandler } from '@/shared/actions/withClientErrorHandler/withClientErrorHandler';

async function executeTabelionatoProtestoDashboardFunilService() {
  return await TabelionatoProtestoDashboardFunilData();
}

export const TabelionatoProtestoDashboardFunilService = withClientErrorHandler(
  executeTabelionatoProtestoDashboardFunilService,
);
